src/build_loss_objective.jl

```diff
@@ -5,6 +5,22 @@ struct DiffEqObjective{F,F2} <: Function
   cost_function2::F2
 end
 
+function diffeq_sen_full(f, u0, tspan, p, t; alg=Tsit5(), kwargs...)
```

Comment:
this isn't used.

Reply:
https://github.com/JuliaDiffEq/DiffEqParamEstim.jl/pull/98/files/51946011756285f36f80c172712a5a0ae6c86460#diff-5d1e39f5dbff55f43e1133fa9262904eR71?

Reply:
it shouldn't be used?